references/cli/utils.go

```diff
@@ -229,3 +229,59 @@ func AskToChooseOneService(services []types.ResourceItem, selectPort bool) (*typ
 	// it should never happen.
 	return nil, 0, errors.New("no service match for your choice")
 }
+
+func convertApplicationRevisionTo(format string, apprev *v1beta1.ApplicationRevision) (string, error) {
```

Comment:
Could you make this function more general? Like `PrettyPrintObject`? And change `apprev *v1beta1.ApplicationRevision`  to `obj client.Object`?

That will make this function easier to be reused.